pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -253,6 +256,19 @@ func (a *Actuator) Delete(ctx context.Context, cluster *clusterv1.Cluster, machi
 
 	defer scope.Close()
 
+	a.log.Info("Cordoning the node", machine.Name)
+	err, patchErr := a.cordon(ctx, cluster, machine, scope)
+	if err != nil {
+		return errors.Errorf("Faild to cordon the node: %+v", err)
+	}
+
+	if patchErr != nil {
+		return errors.Errorf("Faild to cordon the node: %+v", err)
+	}
+
+	// sleep, so I can investigate the state of the node and pods with kubectl
+	time.Sleep(120 * time.Second)
```

Comment:
Based on the comment, I'm assuming this is just for temporary debugging purposes?

Reply:
Exactly, I"m making it sleep for a bit, so I can play around with kubectl to see if everything looks fine.